index.js

```diff
@@ -1,132 +1,146 @@
+  info.htmlBody = body
+  return info
 }
 
-(async () => {
-  const tools = await CheckWebsite.load(process.argv[3])
-  if (process.argv[2] === 'check') {
-    await tools.check()
-  }
-})()
-
-
-
-const csvPreviewLinks = async () => {
-  const dpUrl = `https://pkgstore.datahub.io/core/finance-vix/latest/datapackage.json`
-  const res = await fetch(dpUrl)
+const datapackageJson = async (url) => {
+  const res = await fetch(url)
   let body = await res.text()
   body = JSON.parse(body)
-  for (const idx in body.resources) {
-    const resource = body.resources[idx]
-    if (resource.datahub.type === 'derived/preview') {
-      const previewUrl = resource.path
-      const resPreview = await fetch(previewUrl)
-      return resPreview
-    } 
-  }
-}
-const csvLinks = async () => {
-  const url = 'https://datahub.io/core/finance-vix/r/vix-daily.csv'
-  const res = await fetch(url)
-  return res
-}
-const jsonLinks = async () => {
-  const url = `http://datahub.io/core/finance-vix/r/vix-daily.json`
-  const res = await fetch(url)
-  return res
+  return body
 }
-const pageInfo = async () => {
-  const url = 'http://datahub.io/core/finance-vix'
+
+const resourceLinks = async (url) => {
   const res = await fetch(url)
   return res
 }
 
-const datapackageJson = async () => {
-  const url = 'https://pkgstore.datahub.io/core/finance-vix/latest/datapackage.json'
-  const res = await fetch(url)
-  return res
+const writeToCSV = async (fields, toCsv) => {
+  var csv = json2csv(toCsv) + newLine;
+
+  await fs.appendFile('status.csv', csv, err => {
+      if (err) {
+        console.log(err)
+      }
+      console.log('The data appended to file!');
+  });
 }
 
-module.exports.CheckWebsite = CheckWebsite
+//checkShowcase page returns 200 and get html content
+datasetsToTest.forEach(async dataset => {
+  const statuses = {}
+  const date = new Date()
+  statuses.id = date.toISOString()
+  statuses.name = dataset.name
+  const showcaseUrl = `https://datahub.io/${dataset.owner}/${dataset.name}`
+  
+  // page status 
+  const page = await checkPage(showcaseUrl) //returns {status: statusCode, body: htmlBody}
+  statuses.page_status = page.status
+  if (page.status !== 200) {
+    // exit
+  } else {
+    const $ = cheerio.load(page.htmlBody)
+    let datackageUrl = `https://pkgstore.datahub.io/${dataset.owner}/${dataset.name}/latest/datapackage.json`
+    const dp = await datapackageJson(datackageUrl)
+    
+    // page title
+    let pageTitle = $('head').find('title').text()
+    if (pageTitle.includes(dp.title)) {
+      statuses.page_title = 'OK'
+    } else {
+      statuses.page_title = 'NOT OK'
+    }
+    
+    //dataset title
+    let datasetTitle = $('.showcase-page-header').find('h1').text()
+    if (datasetTitle.includes(dp.title)) {
+      statuses.dataset_title = 'OK'
+    } else {
+      statuses.dataset_title = 'NOT OK'
+    }
+    
+    // readme
+    let readme = $('.inner_container').find('#readme').text()
+    if (readme === 'Read me') {
```

Comment:
Is "Read me" fixed? what if it is READEM or readm etc...

Reply:
@zelima this is fixed